src/test/java/com/openlattice/rehearsal/linking/LinkingFeedbackTest.kt

```diff
@@ -82,6 +82,20 @@ class LinkingFeedbackTest : SetupTestData() {
             )
 
             waitForBackgroundServices()
+
```

Comment:
Why move this chunk from the test case to the initializer?

Reply:
I mean both are fine in my opinion, just asking

Reply:
Spotbugs didn't like writing to a static variable from instance methods and this makes it so that the order of the tests shouldn't matter